packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -1326,6 +1327,28 @@ class DDSMastheadComposite extends HostListenerMixin(LitElement) {
       : unauthenticatedCtaButtons;
   }
 
+  /**
+   * A reference to the dds-masthead element.
+   */
+  @query(`${ddsPrefix}-masthead`)
+  mastheadRef;
+
+  /**
+   * Mutation observer
+   */
+  private _heightMutationObserver = new MutationObserver(
+    this._setContainerHeight.bind(this)
+  );
+
+  /**
+   * Sets root element's height equal to the height of the fixed masthead elements.
+   */
+  protected _setContainerHeight() {
+    const { mastheadRef } = this;
+    this.style.display = 'block';
+    this.style.height = `${mastheadRef.getBoundingClientRect().height}px`;
```

Comment:
Does this line throw a ts error? `mastheadRef` is potentially null as the result of a querySelector.

Reply:
I didn't get a TS error, but you're correct that I could write this more defensively. For now, this method is only ever executed as a callback to the mutation observer that's watching the element `mastheadRef` represents, so I don't _think_ there's an opportunity for it to be null. But I'll add a check anyway 😄